library.json

```diff
@@ -13,7 +13,7 @@
     },
     "repository": {
         "type": "git",
-        "url": "https://github.com/Mnark/esp32-camera.git",

```

Comment:
`https://github.com/Mnark/esp32-camera.git` is valid form. Please revert back

Reply:
Done.